### DISCUSSION TITLE
Welcome to MaslOS Discussions!

### DISCUSSION THREAD

**Discussion by marceldobehere** [marceldobehere/MaslOS#3] [Announcements]
## 👋 Hoi!
  We have the discussions thing now!
  Ya can:
  * Ask questions.
  * Share your suggestions.
  * Talk with other people.
  * Welcome others.
  * Be cool.

  Be sure to introduce yourselves by writing a comment!

**Comment by Yo-yo-ooo** [comment]
How can I contact you？

**Reply by marceldobehere** [reply]
You can send me a friend request on discord. My username is marceldarcel. If you don't gave discord then you can reach me per gmail if you want.

**Reply by Yo-yo-ooo** [reply]
Thanks
